src/reanimated2/__mocks__/NativeReanimated.js

```diff
@@ -2,7 +2,20 @@ global._setGlobalConsole = (val) => {};
 
 export default {
   installCoreFunctions: () => {},
-  makeShareable: (worklet) => worklet,
+  makeShareable: (worklet) => {
+    var _f = function _f() {
+      return worklet();
+    };
+
+    _f._closure = {};
+    _f.asString = worklet.toString();
+    _f.__workletHash = Math.floor(Math.random() * 10000000000);
```

Comment:
1. Why such a number?
2. We definitely don't want to call `Math.random()` here, `__workletHash` should have at least a deterministic and, preferably, a static value.

Reply:
1. Based on my observation, babel transformation results in following structure. `__workletHash` is 14 digits long number. Did not dig deep enough, instead I decided to generate random number on each run. 
2. Agree, long term solution is to generate stable hash for passed function. Anything on your mind? Is it even important for mocking? Not sure.

Reply:
AFAIK `__workletHash` just needs to be unique and the same worklet (looks like a hash is built based on arguments and function content as string) should have the same hash. I'm pretty sure we can use an increasing id for that or hash of `_f.toString()` to be correct.

Reply:
Created a test util for generating "worklet" with stable hash. Same way as plugin does.